### PR TITLE
docs: update release info now that v7 is released

### PR DESCRIPTION
**Pull request by jenniferfell**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

https://angular.io/guide/releases
was written at v6, so the release schedule and support dates are now out-of-date

Issue Number: N/A

## What is the new behavior?

* Release schedule section lists upcoming at v8 and v9
* Support section lists v4-v7
* Minor edits and table rework for consistency and readability

Note: Technically, v6 is still in Active Support time window until Nov 3, but rather than update this doc again then I went ahead and marked it as LTS. In practice, we don't have any plans I know of to put out a v6+ "active support" release now that v7 is released. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [X ] No

### CODE REVIEW

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -65,68 +65,33 @@ Disclaimer: The dates are offered as general guidance and may be adjusted by us
...
-        <td>May 3, 2018</td>
-        <td>November 3, 2018</td>
-        <td>November 3, 2019</td>
-    </tr>
-
-</table>
 
+Version | Status | Released | Active Ends | LTS Ends
+------- | ------ | -------- | ----------- | -------- 
+^7.0.0 | Active | Oct 18, 2018 | Apr 18, 2019 | Apr 18, 2020
```
Comment (IgorMinar) [inline aio/content/guide/releases.md]: can you add whitespace after the text in each column to format this in a human readable way? the extra space will not be visible in the html rendering of this content.

Comment (IgorMinar) [reply aio/content/guide/releases.md]: see the table above in this diff for an example.

**Review by IgorMinar — APPROVED** [review]
lgmt with some formatting nitpicks.

**Comment by preethamvishy** [comment]
I've created a PR for this. Could you check if #26615 is good to go?

File: aio/content/guide/releases.md (hunk 1 above)
Comment (jenniferfell) [reply aio/content/guide/releases.md]: done

**Review by StephenFluin — APPROVED** [review]

**Pull request closed without merging**